### DISCUSSION TITLE
[Plugin Suggestion] Microsoft SQL Server plugin

### DISCUSSION THREAD

**Discussion by hades200082** [dokku/dokku#6605] [Ideas]
Microsoft now provides Docker images for a Linux version of Microsoft SQL Server.

https://learn.microsoft.com/en-us/sql/linux/quickstart-install-connect-docker

I find myself working on .Net Core projects more often these days and there is often a requirement to use MSSQL on these projects.

**Comment by josegonzalez** [comment]
I don't personally have any use for SQL server, but folks are welcome to create their own plugin as a community project.

That said, is this something you'd consider sponsoring? If so, we have a doc on that [here](https://github.com/dokku/.github/blob/master/SPONSORING.md).
